src/web/lib/components/ThemePatternPicker/index.js

```diff
@@ -1,27 +1,38 @@
 import React from "react";
-import classnames from "classnames";
 
 import "./index.scss";
 
 import { colorToCSS } from "../../../../lib/themes";
 import { bgImages } from "../../../../lib/assets";
 import Metrics from "../../../../lib/metrics";
 
+console.log(bgImages);
```

Comment:
Looks like a debugging leftover that should get deleted?

Reply:
Indeed, fixed.